vignettes/calculate_known_outcomes.Rmd

```diff
@@ -0,0 +1,299 @@
+---
+title: "Calculating disease severity by adjusting for delays between onset and outcome during an outbreak"
+output:
+  html_document: 
+    toc: true
+    theme: cosmo
+editor_options:
+  chunk_output_type: console
+subttitle: Estimating the number of cases that likely have a known outcome at each
+  time, for a time series of cases and deaths
+bibliography: resources/library.bib
+csl: resources/bmj.csl
+vignette: "# %\\VignetteIndexEntry{datadelay Case Study 1}\n%\\VignetteEncoding{UTF-8}\n%\\VignetteEngine{knitr::rmarkdown}\n"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  collapse=TRUE,
+  echo=TRUE,
+  comment = "#>",
+  eval=TRUE
+)
+```
+
+# Overview
+This vignette outlines how to use `datadelay` to calculate disease severity during an ongoing outbreak. We wish to estimate either the
+*case fatality risk (CFR)* or *hospitalisation fatality risk (HFR)*. We do so using a method that uses a time series of cases and deaths with a
+delay distribution. The methods used here originate from Nishiura et al. (2009)^[@nishiura2009early]^.
+
+This vignette demonstrates how to implement the core functions within `datadelay`. Specifically, we will cover
+
+1. Why naive estimates of CFR or HFR generated during an ongoing outbreak require adjusting, given out of sync time series are often being compared.
+2. Arrive at estimates over time for the number of symptomatic individuals that will likely be known to have had an outcome $t$ days after
+symptom onset
+3. Define an unbiased estimator to estimate the CFR or HFR from Nishiura et al (2009)^[@nishiura2009early]^
+4. Numerically implement the unbiased estimator numerically using maximum likelihood estimation to arrive at central estimates of the CFR or 
+HFR, along with a confidence interval.
+
+# Setup: installation and requirements
+
+`datadelay` can be installed from Github, using the `devtools` package, with the following command:
+
+```{r message = FALSE, warning = FALSE, eval = FALSE}
+# installs the datadelay package from Github, as recommended on their repository:
+# https://github.com/epiverse-trace/datadelay
+if(!require("devtools")) install.packages("devtools")
+devtools::install_github("epiverse-trace/datadelay")
```

Comment:
Swap to `remotes` for consistency with below (or vice versa)?

Reply:
Fixed in the next PR